inspirehep/dojson/institutions/fields/bd1xx.py

```diff
@@ -75,7 +75,7 @@ def set_value(key, val):
     set_value('obsolete_ICN', value.get('x'))
     set_value('new_ICN', value.get('t'))
 
-    names = list(utils.force_list(value.get('a'))) or []
+    names = list(utils.force_list(value.get('a')) or [])
```

Comment:
Ah, now I understand the bug here: `utils.force_list(None)` raises. This looks like it could happen in a lot of other places.